Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -26,6 +26,7 @@
 #include "config.h"
 #include "WebCoreArgumentCoders.h"
 
+#include "GeneratedSerializers.cpp"
```

Comment:
Not sure whether this is correct, but it was the only way I found to make sure that the `WebCore::TransformOperation::Type` validity code generated in `GeneratedSerializers.cpp` was available to this source file when encoding a transform operation's type.